release/ray_release/tests/test_wheels.py

```diff
@@ -49,20 +49,20 @@ def test_get_ray_wheels_url(remove_buildkite_env):
         repo_url="https://github.com/ray-project/ray.git",
         branch="master",
         commit="1234",
-        ray_version="3.0.0.dev0",
+        ray_version="2.9.0",
```

Comment:
all changes in this file should be reverted.

all these are unit test cases of `ray_release`.